docs/resources/evs_volume.md

```diff
@@ -45,48 +45,55 @@ resource "huaweicloud_evs_volume" "volume" {
 
 The following arguments are supported:
 
-* `region` - (Optional, String, ForceNew) The region in which to create the EVS volume resource. If omitted, the provider-level region will be used. Changing this creates a new EVS resource.
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the EVS volume resource.
+    If omitted, the provider-level region will be used.
+    Changing this creates a new EVS volume resource.
 
-* `availability_zone` - (Required, String, ForceNew) The availability zone for the volume.
-    Changing this creates a new volume.
+* `availability_zone` - (Required, String, ForceNew) Specifies the availability zone for the volume.
+    Changing this creates a new EVS volume resource.
 
 * `volume_type` - (Required, String, ForceNew) Specifies the disk type.
     Currently, the value can be SSD, SAS, or SATA.
     - SSD: specifies the ultra-high I/O disk type.
     - SAS: specifies the high I/O disk type.
-    - SATA: specifies the common I/O disk type.
+    - GPSSD: specifies the common I/O disk type.
```

Comment:
the description of `GPSSD` is wrong, please ref to https://support.huaweicloud.com/intl/en-us/api-evs/evs_04_2013.html